_security/configuration/best-practices.md

```diff
@@ -0,0 +1,127 @@
+2. Regular review: Regularly review and audit assigned roles to ensure alignment with organizational policies and access needs.
+
+For more information about roles, visit the documentation on [defining users and roles in OpenSearch]({{site.url}}{{site.baseurl}}/security/access-control/users-roles/).
+
+## 5. Verify DLS, FLS and field masking
+
+If you have configured Document Level Security (DLS), Field Level Security (FLS), or field masking, make you double-check your role definitions, especially if a user is mapped to multiple roles. It is highly recommended to test this by making a GET request to `_plugins/_security/authinfo`.
+
+Detailed examples and additional configurations are available at:
+ - [Document-level security]({{site.url}}{{site.baseurl}}/security/access-control/document-level-security/)
+ - [Field-level security]({{site.url}}{{site.baseurl}}/security/access-control/field-level-security/)
+ - [Field masking]({{site.url}}{{site.baseurl}}/security/access-control/field-masking/)
+
+## 6. Strip audit logging configuration to essentials only
+
+Extensive audit logging can degrade system performance due to the following reasons:
+- Each logged event adds to the processing load.
+- Audit logs can quickly grow in size, consuming significant disk space.
+
+To ensure optimal performance, disable unnecessary logging and be selective about which logs are used. If not strictly required by compliance regulations, consider turning off audit logging. If audit logging is essential for your cluster, configure it according to your compliance requirements.
+
+Whenever possible, adhere to these recommendations:
+
+- Set `audit.log_request_body` to `false`.
+- Set `audit.resolve_bulk_requests` to `false`.
+- Enable `compliance.write_log_diffs`.
+- Minimize entries for `compliance.read_watched_fields`.
+- Minimize entries for `compliance.write_watched_indices`.
+
+## 7. Explore disabling private tenant
+
+In many cases, the use of private tenants is unnecessary, yet this feature is enabled by default. As a result, every OpenSearch Dashboards user is provided with their own private tenant and a corresponding new index to save objects. This can lead to a large number of unnecessary indexes. Evaluate whether this feature is needed, and if it isn't, disable it by adding the following configuration to the `config.yml` file:
+
+```yaml
+config:
+  dynamic:
+    kibana:
+      multitenancy_enabled: true
+      private_tenant_enabled: false
+```
+
+## 8. Manage the configuration via securityadmin.sh
+
+Use `securityadmin.sh` to manage the configuration of your clusters. `securityadmin.sh` is a command-line tool provided by OpenSearch for managing security configurations. It allows administrators to efficiently manage security settings, including roles, role mappings, and other security-related configurations within an OpenSearch cluster.
```

Comment:
Should `securityadmin.sh` also be in code font in the heading here and below?

Reply:
Yes thanks Nate done :)